web/helpers/project_helper.rb

```diff
@@ -2,4 +2,8 @@ module ProjectHelper
   def current_project
     @current_project ||= Projects::Find.new(params[:project_name], current_user).run!
```

Comment:
Not sure this one is still valid because we don't have a `project_name`, we have a `owner` and a `repo`, isn't it?